src/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+require("dotenv").config({ path: "./config.env" });
+const port = process.env.PORT || 3000;
+app.use(cors());
+app.use(express.json());
+
+
+
+//router
+app.get("./", (req,res)=> {
```

Comment:
Don't use '.' in route names, this is for a web server not your local file system :)